vulkan_generator/generator.py

```diff
@@ -57,8 +61,32 @@ def print_vulkan_metaadata(vulkan_metadata: types.AllVulkanTypes) -> None:
     pretty_printer.pprint(vulkan_metadata.funcpointers)
 
 
-def generate(vulkan_xml_path: Path) -> bool:
+def basic_generate(target: str,
+                   output_dir: Path,
+                   all_vulkan_types: types.AllVulkanTypes,
+                   generate_header,
+                   generate_cpp,
+                   generate_test):
+
+    generate_header(os.path.join(output_dir, target + ".h"), all_vulkan_types)
+    generate_cpp(os.path.join(output_dir, target + ".cc"), all_vulkan_types)
+    generate_test(os.path.join(output_dir, target + "_tests.cpp"), all_vulkan_types)
```

Comment:
This still generates names based on the `target` string, which does not follow the naming from the style guide.

Reply:
fixed